src/Operation/Find.php

```diff
@@ -347,13 +347,22 @@ public function execute(Server $server)
         return $cursor;
     }
 
+    /**
+     * Returns the command document for this operation.
+     *
+     * @see Explainable::getCommandDocument()
+     * @param Server $server
+     * @return array
+     */
     public function getCommandDocument(Server $server)
     {
         return $this->createCommandDocument();
     }
 
     /**
      * Construct a command document for Find
+     *
+     * @return array
```

Comment:
Was this preferable to adding an `array` return type hint?

I realize we probably can't do that for `Explainable::getCommandDocument` since it's an interface method, but this is private.

Reply:
There might be more instances like this, but I've fixed this one already. Usually, the type sniffs would suggest using a native return type over `@return` in this case, and even fix it automatically. However, I have disabled this sniff in the `src` folder because of the BC breaks it would cause in public methods. If I were able to only target private members, we could fix this automatically.

Note that with PHP 8.1, the new `#[ReturnTypeWillChange]` attribute allows developers to mark methods that will have a return type in a future major release (to indicate that downstream classes should add the type before then). As part of our 8.1 compatibility effort, we will most likely be annotating those methods with the attribute above, at which point we can also add a suppression for that coding standards violation and turn on the automatic sniff, which will result in these being fixed automatically.